Makefile

```diff
@@ -3,8 +3,8 @@ HOSTNAME=registry.terraform.io
 NAMESPACE=l-with
 NAME=ldap
 BINARY=terraform-provider-${NAME}
-VERSION=9.9.9
-OS_ARCH=darwin_arm64
+VERSION?=0.10.1
```

Comment:
I like to use a fixed version never used in "production" for local testing.

Reply:
ahh, make sense! reverted back to 9.9.9